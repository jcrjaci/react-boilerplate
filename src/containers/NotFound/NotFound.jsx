import React from 'react';
import { Link } from 'react-router-dom';
import './NotFound.scss';


const NotFound = () => (
  <div className="not-found">
    <div className="message">
      <h1>
          Page Not Found
      </h1>
    </div>
    <Link to="/">
      Take me back to home
    </Link>
  </div>
);

export default NotFound;
