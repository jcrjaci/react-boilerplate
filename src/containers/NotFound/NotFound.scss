.not-found {
  align-items: center;
  display: flex;
  flex-direction: column;
  height: 100%;
  justify-content: center;

  .message {
    align-items: center;
    display: flex;
    flex: 1;
  }
  
  a {
    color: #000;
    text-decoration: none;
  }
  
  a.hover {
    cursor: pointer;
  }
}