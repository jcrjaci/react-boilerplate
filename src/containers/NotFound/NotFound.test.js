import NotFound from './NotFound';

describe('Loading component', () => {
  it('should render Not found component', () => {
    expect(NotFound()).toMatchSnapshot();
  });
});
