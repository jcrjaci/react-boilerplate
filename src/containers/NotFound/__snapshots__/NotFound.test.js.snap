// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render Not found component 1`] = `
<div
  className="not-found"
>
  <div
    className="message"
  >
    <h1>
      Page Not Found
    </h1>
  </div>
  <a
    href="/"
  >
    Take me back to home
  </a>
</div>
`;
